components/event-feed-service/pkg/persistence/feed-elastic.go

```diff
@@ -267,28 +232,29 @@ func (efs ElasticFeedStore) getCounts(q *util.FeedSummaryQuery) (map[string]int6
 
 func newBoolQueryFromFilters(filters map[string][]string) *olivere.BoolQuery {
 	boolQuery := olivere.NewBoolQuery()
-	field := "tags" // all filter terms are in the "tags" field
-
-	for _, values := range filters {
-		if len(values) > 0 {
-			if len(values) > 1 {
-				// if the same key is associated with multiple values, construct an OR query (terms query)
-				termsQuery := olivere.NewTermsQuery(field, util.StringArrayToInterfaceArray(values)...)
-				boolQuery = boolQuery.Must(termsQuery)
-			} else {
-				// if there's only one value, construct an AND query (one MUST with multiple values for "tags"), e.g.:
-				// "must":[
-				//    {"term":{"tags":"org_1"},
-				//	  {"term":{"tags":"org_2"}
-				//  ]
-				value := values[0]
-				boolQuery.Must(olivere.NewTermQuery(field, value))
-			}
+
+	for field, values := range filters {
```

Comment:
Change the filtering to filter off a specific field for each type of filter and not only the tags field.